client/gatsby-ssr.js

```diff
@@ -32,3 +32,19 @@ export const onRenderBody = ({
   setHeadComponents(getheadTagComponents());
   setPostBodyComponents(getPostBodyComponents(pathname));
 };
+
+export const onPreRenderHTML = ({
+  getHeadComponents,
+  replaceHeadComponents
+}) => {
+  const headComponents = getHeadComponents();
+  headComponents.sort((x, y) => {
+    if (x.key === 'boostrap-min-preload' || x.key === 'boostrap-min') {
+      return -1;
+    } else if (y.key === 'boostrap-min-preload' || y.key === 'boostrap-min') {
```

Comment:
```suggestion
    if (x.key === 'bootstrap-min-preload' || x.key === 'bootstrap-min') {
      return -1;
    } else if (y.key === 'bootstrap-min-preload' || y.key === 'bootstrap-min') {
```

Reply:
Good catch.  The original keys were misspelled. I updated all of the accordingly.